SyncApp/Logic/CountriesLogic.cs

```diff
@@ -51,5 +76,37 @@ internal bool CheckIfHasValues(Countries country)
         {
             return !string.IsNullOrEmpty(country?.BranchCode) && !string.IsNullOrEmpty(country?.CustomerCode) && country?.CountryTax != null;
         }
+
+        public async Task<List<Country>> GetCountriesAsync()
```

Comment:
To Differentiate it from DB function in the future can you please rename it to "GetShopifyCountries" as to understand its from shopify not DB

Reply:
Done